sdk/streamanalytics/Azure.ResourceManager.StreamAnalytics/api/Azure.ResourceManager.StreamAnalytics.netstandard2.0.cs

```diff
@@ -132,7 +132,7 @@ public partial class StreamingJobCollection : Azure.ResourceManager.ArmCollectio
     public partial class StreamingJobData : Azure.ResourceManager.Models.TrackedResourceData
     {
         public StreamingJobData(Azure.Core.AzureLocation location) : base (default(Azure.Core.AzureLocation)) { }
-        public Azure.Core.ResourceIdentifier ClusterId { get { throw null; } set { } }
+        public string ClusterId { get { throw null; } set { } }
```

Comment:
This change should be introduced by the new `no-property-type-replacement` configuration, we need to verify if it is necessary to add that

Reply:
Does this change mean this is not a resource identifier? If so we should change the type here back

Reply:
fixed